components/src/web-components/gs-app.ts

```diff
@@ -59,6 +60,25 @@ export class AppComponent extends LitElement {
         aminoAcidPositions?: string[];
     }[] = [];
 
+    /**
+     * Supply a link template for nucleotide and amino acid mutations.
+     * The template should include '{{mutation}}' where the mutation code will be inserted, for example:
+     *
+     *     https://my-site.org/query?nucleotideMutation={{mutation}}
+     */
+    @provide({ context: mutationLinkTemplateContext })
+    @property({ type: Object })
+    mutationLinkTemplate: MutationLinkTemplate = {
```

Comment:
I think we should spell that type out so that it shows up usefully in storybook. Currently, someone who doesn't look at our code will not understand the type: 
<img width="1013" height="176" alt="Image" src="https://github.com/user-attachments/assets/0b0b0d9f-548a-42a7-a58e-c9b671e6e9e0" />


```suggestion
    mutationLinkTemplate: { nucleotideMutation: string | undefined, aminoAcidMutation: string l undefined } = {
```

Reply:
https://github.com/GenSpectrum/dashboard-components/pull/995/commits/2c24df2c9c45a8cc7333c37c408f77fdca996f32